--- ksvg/configure.in.old	2013-10-17 02:28:44.070536104 +0100
+++ ksvg/configure.in.in	2013-10-17 02:31:37.241196471 +0100
@@ -253,31 +253,11 @@
 AC_SUBST(LCMS_LIBS)
 
 # Check for fribidi
-KDE_FIND_PATH(fribidi-config, FRIBIDI_CONFIG, [${prefix}/bin ${exec_prefix}/bin /usr/local/bin /opt/local/bin],)
-
-if test -n "$FRIBIDI_CONFIG"; then
-  vers=`$FRIBIDI_CONFIG --version 2>/dev/null | awk 'BEGIN { FS = "."; } { printf "%d", ($1 * 1000 + $2) * 1000 + $3;}'`
-  if test -n "$vers" && test "$vers" -ge 10004
-  then
-     FRIBIDI_LIBS="`$FRIBIDI_CONFIG --libs`"
-     FRIBIDI_RPATH=
-     for args in $FIBIDI_LIBS; do
-	  case $args in
-	    -L*)
-  	       FRIBIDI_RPATH="$FRIBIDI_CONFIG $args"
- 	       ;;
-          esac
-     done
-     FRIBIDI_RPATH=`echo $FRIBIDI_RPATH | sed -e "s/-L/-R/g"`
-     FRIBIDI_CFLAGS="`$FRIBIDI_CONFIG --cflags`"
-
-     AC_DEFINE_UNQUOTED(HAVE_FRIBIDI, 1, [Defines if your system has the fribidi library])
-  fi
-fi
-
+PKG_CHECK_MODULES(FRIBIDI, fribidi,
+  AC_DEFINE_UNQUOTED(HAVE_FRIBIDI, 1, [Defines if your system has the fribidi library])
+)
 AC_SUBST(FRIBIDI_LIBS)
 AC_SUBST(FRIBIDI_CFLAGS)
-AC_SUBST(FRIBIDI_RPATH)
 
 if test -z "$FRIBIDI_LIBS"; then
 	DO_NOT_COMPILE="$DO_NOT_COMPILE ksvg"
